feat: add option to not fail on failing test suite (#4771)
mochajs/mocha: docs/index.md

- Try something like [wtfnode][npm-wtfnode]
- Use [`.only`](#exclusive-tests) until you find the test that causes Mocha to hang

### `--pass-on-failing-test-suite`

> If set on true, Mocha returns zero code even if there are failed tests during run.

Reviewer: [Docs] Nit: standardizing to other docs...

```suggestion
If set to `true`, Mocha returns exit code `0` even if there are failed tests during run.
```
